.github/workflows/nodejs.yml

```diff
@@ -4,23 +4,43 @@ on: [push]
 
 jobs:
   build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
 
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Build and Test
+        run: |
+          yarn
+          yarn build
+          yarn test
+        env:
+          CI: true
+  build-examples:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
         node-version: [8.x, 10.x, 12.x]
 
     steps:
-    - uses: actions/checkout@v1
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - name: Build and Test
-      run: |
-        yarn
-        yarn build
-        yarn test
-      env:
-        CI: true
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Build Examples
+        run: |
+          yarn
+          yarn build
+          cd $GITHUB_WORKSPACE/examples
+          for dir in $GITHUB_WORKSPACE/examples/*; do (cd "$dir"; yarn; yarn clean; yarn build; cd ../); done
```

Comment:
Here's where we loop through each example to ensure it still compiles. Is there potentially an easier / more efficient way to do this?